src/app/core/inventory/inventory.service.ts

```diff
@@ -38,14 +38,13 @@ export class InventoryService {
     };
   }
 
-  getInventoryAvailability({id, startDate, endDate, zoneName, flightId}): Observable<Availability> {
+  getInventoryAvailability({id, startDate, endDate, zoneName}): Observable<Availability> {
     return this.augury.follow('prx:inventory', {id}).pipe(
       switchMap(inventory => {
         return inventory.follow('prx:availability', {
           startDate,
           endDate,
-          zoneName,
-          flightId
+          zoneName
```

Comment:
shouldn't need the fightId to query availability

Reply:
Yep,  you should only need that flightId if you want to see the allocation sums for _that_ flight.

Reply:
hmm, maybe I am confused. I took out flightId here because I was working toward being able to see the availability for flights that were not yet created. (I had just given it all the params before without thinking about uncreated flights I guess.) Should I include the flightId for availability when the flight has been created? Is it giving me a sum of all flights otherwise? Or maybe you're just talking allocations, which is my next step. This part is still just working with the availability.

Reply:
Yep,  for flights that are created, you would want to include that flightId.  But obviously, for for flights not persisted, then you can't include it.

Sort of a trick of formatting.  In the availability table this means the day is maxed out (no more inventory is available):
<img width="278" alt="Screen Shot 2019-10-16 at 2 49 39 PM" src="https://user-images.githubusercontent.com/329012/66953804-c8cedd80-f024-11e9-8ee1-defadd0733ac.png">

In this case the api response (given flightId) should be something like
```
    {
      "allocated": 5000,
      "availability": 0,
      "date": "2019-04-14"
    },
```
But here is an example where there is 1000 more we could add:
<img width="275" alt="Screen Shot 2019-10-16 at 2 49 45 PM" src="https://user-images.githubusercontent.com/329012/66953759-b359b380-f024-11e9-9931-bd5aecd553b5.png">

In this case the api (given flightId) might be something like
```
    {
      "allocated": 5000,
      "availability": 1000,
      "date": "2019-04-14"
    },
```
Or in the case we supply no flight id:
```
    {
      "allocated": <some number of allocations summed, from other flights>,
      "availability": 1000,
      "date": "2019-04-14"
    },
```
In this last case, you'd have to hit the allocation preview endpoint to see what is allocated on that day (hypothetically).

Reply:
Ohhhh... I think I get it now, thank you. I was thinking I would be hitting the allocation preview endpoint for all flights, but it's just for the flights that are not created yet.